pepper-apis/src/main/resources/db-changes/schema/DDP-2565-modify-study-settings.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <changeSet id="20200623-modify-study-settings" author="sampath">
+
+        <addColumn tableName="study_settings">
```

Comment:
Debated to add these study config columns to "umbrella_study" OR "study_settings" .  
Since study_settings is meant for optional configuration was little hesitant to add to study_settings rather than umbrella_study.